main/CheckBox/index.tsx

```diff
@@ -1,20 +1,105 @@
-import React, { FC } from 'react';
+import React, { FC, useCallback, useState } from 'react';
 import {
-  Text,
-  View,
+  TouchableHighlight,
 } from 'react-native';
 
-interface Props {
-  defaultChecked?: boolean,
-  disabled?: boolean,
+import styled from 'styled-components/native';
+
+type MarkerContainerProps = {
```

Comment:
`Interface` is better than to use `type`. This is because the interface is more flexible in scalability.

Reply:
I looked at the review and searched for the difference between type and interface. And the link below gave us a better understanding of what the reviewer(@JeffGuKang) intended.

- https://www.typescriptlang.org/docs/handbook/advanced-types.html#interfaces-vs-type-aliases